File: WebServiceSchoopy/src/java/pkgData/Student.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package pkgData;

/**
 *
 * @author schueler
 */
public class Student extends Person{
    private Room visitedClass;

    public Student(Room visitedClass, String firstName, String lastName, String schoolemail, String username, String password) {
        super(firstName, lastName, schoolemail, username, password);
        this.visitedClass = visitedClass;
    }

    
    public Student(){
    }
    
    public Student(String username,String password){
        this.username = username;
        this.password = password;
    }
    
        
    public Student(String firstName, String lastName, String schoolemail, String username, String password) {
        super(firstName, lastName, schoolemail, username, password);
    }

    public Room getVisitedClass() {
        return visitedClass;
    }

    public void setVisitedClass(Room visitedClass) {
        this.visitedClass = visitedClass;
    }
    
    
}

File: SchoopyC#/Schoopy/Data/Student.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Schoopy.Data
{
    public class Student : Person
    {
        public Room visitedClass { get;  set; }
        public Student(string _username, string _passwort , Room _visitedClass)
        {
            this.password = _passwort;
            this.username = _username;
            this.visitedClass = _visitedClass;
        }

        public Student()
        {
        }
    }
}
